src/java.base/share/classes/java/util/stream/Stream.java

```diff
@@ -407,8 +407,8 @@
      *     }
      *
      *     public static void main(String[] args) {
-     *         Stream<Object> stream = ...;
-     *         Stream<Object> expandedStream = stream.mapMulti(C::expandIterable);
+     *         var nestedList = ...;
```

Comment:
I think it would be good to expand the RHS to use what you use in the test, namely

```
var nestedList = List.of(1, List.of(2, List.of(3, 4)), 5);
```

That would clarify the example quite a bit. Similar for the numbers example above.

Reply:
Sounds good. I've added that to the example now. See 944c7d5